.github/workflows/deploy-staging.yml

```diff
@@ -48,31 +56,32 @@ jobs:
           HUSKY: 0
           REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/staging/latest/v3/modern/plugins'
           BUGSNAG_API_KEY: ${{ secrets.RS_STAGING_BUGSNAG_API_KEY }}
-          BUGSNAG_RELEASE_STAGE: 'development'
+          BUGSNAG_RELEASE_STAGE: 'staging'
         run: |
           npm run setup:ci
 
-      - name: Build assets
+      - name: Build files
         env:
           REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/staging/latest/v3/modern/plugins'
           BUGSNAG_API_KEY: ${{ secrets.RS_STAGING_BUGSNAG_API_KEY }}
-          BUGSNAG_RELEASE_STAGE: 'development'
+          BUGSNAG_RELEASE_STAGE: 'staging'
         run: |
           npm run build:browser
           npm run build:browser:modern
 
-      - name: Build LH assets
-        env:
-          REMOTE_MODULES_BASE_PATH: 'https://cdn.rudderlabs.com/staging/latest/v3/modern/plugins'
-          BUGSNAG_API_KEY: ${{ secrets.RS_STAGING_BUGSNAG_API_KEY }}
-          BUGSNAG_RELEASE_STAGE: 'development'
-          BUNDLED_PLUGINS: 'StorageEncryptionLegacy,XhrQueue'
+      - name: Sync files to S3 v3 staging folder
+        run: |
+          aws s3 cp packages/analytics-js/dist/cdn/legacy/iife/ s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/staging/latest/v3/legacy/ --recursive --cache-control max-age=3600
+          aws s3 cp packages/analytics-js/dist/cdn/modern/iife/ s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/staging/latest/v3/modern/ --recursive --cache-control max-age=3600
+          aws s3 cp packages/analytics-js-plugins/dist/cdn/modern/plugins/ s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/staging/latest/v3/modern/plugins/ --recursive --cache-control max-age=3600
+          aws s3 cp packages/analytics-js-integrations/dist/cdn/legacy/js-integrations/ s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/staging/latest/v3/legacy/js-integrations/ --recursive --cache-control max-age=3600
+          aws s3 cp packages/analytics-js-integrations/dist/cdn/modern/js-integrations/ s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/staging/latest/v3/modern/js-integrations/ --recursive --cache-control max-age=3600
+
+      - name: Create Cloudfront invalidation for v3
         run: |
-          export BUNDLED_PLUGINS='StorageEncryptionLegacy,XhrQueue'
-          npm run clean:cache
-          cd packages/analytics-js
-          npm run build:browser:LH
```

Comment:
Let's not remove it. Comment out this part for future reference.

Reply:
Why keep it if no one uses it? Moreover, we should keep customer-specific information outside of the repo.

We can always refer to the commit history if you suggest it'll be helpful later.